docs/docs/settings/spacing-rules.md

```diff
@@ -686,7 +686,14 @@ All headings have a blank line both before and after (except where the heading i
 | `Bottom` | Insert a blank line after headings | N/A | `true` |
 | `Empty Line Between YAML and Header` | Keep the empty line between the YAML frontmatter and header | N/A | `true` |
 
+### Additional Info
 
+!!! Warning
+    If [paragraph blank lines](#paragraph-blank-lines) is on, a newline will still be added between a heading and a paragraph, even if `Bottom` is `false`. You can override this by adding a [custom regex replacement rule](./custom-rules.md#custom-regex-replacement) with the following settings:
+    
+    | regex to find  | flags | regex to replace |
+    |:-------------  |:----- |:---------------- |
+    | `(^#+\s.*)\n+` | `gm`  | `$1\n`           |
```

Comment:
This is a generated file, so this info actually needs to be placed in a different file. You need to add a new file in [here ](https://github.com/platers/obsidian-linter/tree/master/docs/additional-info/rules)for the appropriate rule (file name is the rule identifier/alias like `paragraph-blank-lines`) and  then add this content there.

Reply:
Right, hands working too quickly for the brain again. Should be fixed now.